benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -520,7 +522,7 @@ msgid "enrollment.pages.index.title"
 msgstr "Connect your card"
 
 msgid "enrollment.pages.index.headline"
-msgstr "We have located you in our system!"
+msgstr "Your eligibility is confirmed!<br/>You’re nearly there."
```

Comment:
Design calls for this now:
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/3673236/195220667-10aee3f4-b04d-4c2c-9925-ab387584d7d9.png">

If you take out `<br/>` from the code, and use `col-lg-5` for the width of the parent div of the `h1`, you can achieve this design.

It would require overriding `main_content` of `base.html` _or_ modifying `base.html` so it's easier to override this part:

```html
            <div class="col-lg-10">
              <h1>{{ page.headline }}</h1>
            </div>
```

Reply:
Thanks for the feedback on this and on the [help text classes](https://github.com/cal-itp/benefits/pull/1013#issuecomment-1275398879). I'll work on making those changes

Reply:
Done in 08e9a19a8aeae1609765beca472ad28fd38715d9

### Mobile
The headline is all one line (no break in between). At most mobile widths, the first sentence does not wrap.

![image](https://user-images.githubusercontent.com/25497886/195388127-fa4027e8-8fc0-4340-b092-d817da98de29.png)

Once the width is small enough, it does wrap.   

<img src="https://user-images.githubusercontent.com/25497886/195388198-1df84e80-ea71-4858-9da0-c9e5e74dd2fd.png" width="450">

### Desktop
<img src="https://user-images.githubusercontent.com/25497886/195387975-e7ef8783-ba3f-4df6-b2ea-f6655a9e985e.png" width="450">